docs/includes/usage-examples/UpdateOneTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
```

Comment:
@GromNaN: more generally I don't see a clear benefit to putting PHPUnit test files outside of the `tests/` directory like this. This differs from the PHPLIB approach, where we run example/docs scripts from [ExamplesTest](https://github.com/mongodb/mongo-php-library/blob/master/tests/ExamplesTest.php) and assert output.

Based on the current PHPUnit configuration, these files won't be picked up by the test suite (nor tools like phpcs and phpstan). It might make more sense to organize these files within the actual test suite and then have docs refer to them there. And in that case, we can use meaningful namespaces.

Reply:
The namespace is of no particular interest. I think we'll come back to it to clean it up.